pkg/crc/machine/start.go

```diff
@@ -548,10 +564,6 @@ func createHost(api libmachine.API, machineConfig config.MachineConfig) error {
 	if err := cluster.GenerateKubeAdminUserPassword(); err != nil {
 		return errors.Wrap(err, "Error generating new kubeadmin password")
 	}
-	if err := copyKubeconfig(machineConfig.Name, machineConfig); err != nil {
```

Comment:
If you remove this, there is a small amount of time during `crc start` where `crc status` will be broken.
The machine will be marked as created but the kubeconfig file will be absent.